bin/dmarc_send_reports

```diff
@@ -241,7 +241,7 @@ sub get_smtp_connection {
             if ( $rid ) {
                 my $errors = scalar $$agg_ref->metadata->error;
                 if ( $errors >= 12 ) {
-                    print "Report $rid deleted (too many errors)\n";
+                    print "Report $rid deleted (too many errors)\n" if $verbose;
```

Comment:
Should it still be silent when errors are encountered?

Reply:
I think so, typically these are being run by cron and it's just adding noise. The "too many errors" thing is really a warning that a report couldn't be sent, not a program error.